runtime/src/bank.rs

```diff
@@ -3856,21 +3858,48 @@ impl Bank {
                                 loaded_accounts_count: loaded_accounts.len(),
                                 loaded_accounts_data_size,
                             },
+                            post_accounts_states: loaded_accounts,
+                        })
+                    }
+                    ProcessedTransaction::FeesOnly(fees_only_tx) => {
+                        let loaded_accounts_count = fees_only_tx.rollback_accounts.count();
+                        let post_accounts_states = {
+                            match fees_only_tx.rollback_accounts {
+                                RollbackAccounts::FeePayerOnly {
+                                    fee_payer_account,
+                                    fee_payer_address,
+                                } => {
+                                    vec![(fee_payer_address, fee_payer_account)]
+                                }
+                                RollbackAccounts::SameNonceAndFeePayer { nonce } => {
+                                    let NonceInfo { address, account } = nonce;
+                                    vec![(address, account)]
+                                }
+                                RollbackAccounts::SeparateNonceAndFeePayer {
+                                    nonce,
+                                    fee_payer_account,
+                                    fee_payer_address,
+                                } => {
+                                    let NonceInfo { address, account } = nonce;
+                                    vec![(fee_payer_address, fee_payer_account), (address, account)]
+                                }
```

Comment:
&lt;bikeshedding&gt;

I personally also prefer deconstructing values as well.
But in this case, using field access produces a code that seems slightly more readable:

```suggestion
                                RollbackAccounts::SameNonceAndFeePayer { nonce } => {
                                    vec![(nonce.address, nonce.account)]
                                }
                                RollbackAccounts::SeparateNonceAndFeePayer {
                                    nonce,
                                    fee_payer_account,
                                    fee_payer_address,
                                } => vec![
                                    (fee_payer_address, fee_payer_account), 
                                    (nonde.address, nonce.account)
                                ],
```